lib/Caxy/HtmlDiff/AbstractDiff.php

```diff
@@ -433,7 +433,7 @@ protected function convertHtmlToListOfWords(string $text) : array
         // Normalize no-break-spaces to regular spaces
         $text = str_replace("\xc2\xa0", ' ', $text);
 
-        preg_match_all('/<.+?>|[^<]+/mu', $text, $sentencesAndTags, PREG_SPLIT_NO_EMPTY);
+        preg_match_all('/<[^>]+?>|[^<]+/mu', $text, $sentencesAndTags, PREG_SPLIT_NO_EMPTY);
```

Comment:
Alternatively we could add the `s` (`PCRE_DOTALL`) modifier so that the dot would match newlines as well.

However, this might seem more clear? Are there situations where this change would actually affect the outcome in a negative way? 

I think it's technically allowed to have `>` within an attribute value of an HTML tag...

```html
<div data-some-value="1 > 0">
```

that's technically valid I think?

So are we better off adding the `s` modifier to solve this instead?

Reply:
I've tested all regexes (current, mine, yours) with the example you provided and they all break the tag in the `data-some-value` contents. So this is currently a problem and i dont know how to fix this particular issue.

However, i do agree the dotall modifier is more clear, i will change the merge request to use that.